app/controllers/v2/security_guides_controller.rb

```diff
@@ -6,23 +6,23 @@ class SecurityGuidesController < ApplicationController
     def index
       render_json security_guides
     end
-    permission_for_action :index, Rbac::COMPLIANCE_VIEWER
```

Comment:
What about reusing this constant instead? I think that should lead to a cleaner solution, semantically as well.

Reply:
Good point.